### PR TITLE
White Lotus Surf Camp by Elsa Carlström

### PR DESCRIPTION
**Pull request by elsisco**
Closed my last pull request because of updated code. More info in the readme-file!

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -3,28 +3,113 @@
...
+    <script src="script.js" defer></script>
+    
+    
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    <header>
+      <img class="logo" src="./logo/surf_logo.svg" alt="logo">
```
Comment (HelenaW86) [inline code/index.html:15]: nice that you filled in the alt tag for the logo

File: code/index.html [hunk 2]
```diff
@@ -3,28 +3,113 @@
...
+            </div>
+          </div>    
+        </section>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+    </div>
+    <footer>WHITE LOTUS TRAVELS 2021</footer>
   </body>
 </html>
```
Comment (HelenaW86) [inline code/index.html:115]: Really nice structure at everything=)

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,276 @@
...
+  }
+
+  .input-container {
+    width: 49%;
+  }
+
+  .accordion-container {
+    width: 40%;
+    padding: 47px 48px 0px 20px;
+  }
```
Comment (HelenaW86) [inline code/style.css:275]: Really really nice, clear code that is easy to read and understand

File: code/script.js [hunk 4]
```diff
@@ -1,9 +1,14 @@
...
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
+const questions = document.querySelectorAll(".question");
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+questions.forEach(question => {
```
Comment (HelenaW86) [inline code/script.js:3]: smart to use a forEach function

**Review by HelenaW86 — COMMENTED** [review]
Elsa! I really enjoyed looking at your code! It´s clean and structured. I can´t find anything that I would like to change. Very very nice! =D

File: code/index.html (hunk 2 above)
Comment (elsisco) [reply code/index.html:115]: Thank you so much for the review! :)

**Pull request closed without merging**